demo/handoff/corkboard-assay.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2019 Agoric, under Apache License 2.0
+
+import harden from '@agoric/harden';
+import {
+  mustBeComparable,
+  mustBeSameStructure,
+  sameStructure,
+} from '../../collections/sameStructure';
+import { insist } from '../../collections/insist';
+import { makePrivateName } from '../../collections/PrivateName';
+
+// A CorkboardAssay vouches for Corkboards, which allow vats to connect to one
+// another in a lightweight way. Corkboards can not be combined or split.
+// The quantity must be a unique string. The empty value may be useful for empty
+// purses.
+function makeCorkboardAssayMaker(descriptionCoercer = d => d) {
```

Comment:
I think you were saying that the makeUniAssayMaker was mostly what the CorkboardAssayMaker needed, is this the same thing as the makeUniAssayMaker with some of the names changed, or were there differences in implementation?

Reply:
There are only tiny differences between them. Set operations basically aren't meaningful for corkboards, so includes() returns false unless the two parameters are the same.  I also added one clause to coerce() to support cases where the quantity in the alleged amount has already been registered. 

I'm not sure whether that's correct.